util/test/README.md

```diff
@@ -45,6 +45,8 @@ The demos project contains helper libraries, so the best way to get started is t
 
 Similarly for tests, which often come 1:1 with demos, you can copy-paste an existing test and add your own checks.
 
+When adding a test that needs to compare against a reference image, first run the first without a reference image. It will output the image that it would compare against. Run this image through pngcrush (version 1.7.85 preferred) to reduce repository bloat.
```

Comment:
Rather than specifying the version, which could get stale, better to say that pngcrush must be run with parameters to force RGBA output (I think that means `-noreduce`). Or we could fix our loader code to expand grayscale to RGBA, but this seems easier.